crates/router/src/routes/app.rs

```diff
@@ -1405,7 +1405,27 @@ impl Poll {
 
 pub struct ApiKeys;
 
-#[cfg(feature = "olap")]
+#[cfg(all(feature = "v2", feature = "olap", feature = "merchant_account_v2"))]
+impl ApiKeys {
+    pub fn server(state: AppState) -> Scope {
+        web::scope("/v2/api_keys")
+            .app_data(web::Data::new(state))
+            .service(web::resource("").route(web::post().to(api_key_create)))
+            .service(web::resource("/list").route(web::get().to(api_key_list)))
+            .service(
+                web::resource("/{key_id}")
+                    .route(web::get().to(api_key_retrieve))
+                    .route(web::post().to(api_key_update))
```

Comment:
Shouldn't this be `PUT` method to maintain consistency with the other v2 update APIs?

```suggestion
                    .route(web::put().to(api_key_update))
```